test/puppetlabs/puppetdb/command_test.clj

```diff
@@ -866,7 +866,7 @@
 
 (deftest replace-facts-bad-payload
   (let [bad-command {:command (command-names :replace-facts)
-                     :version 5
+                     :version (apply max (get supported-command-versions "replace facts"))
```

Comment:
@wkalt yeah, I agree that this looks kind of messy right now, but wasn't sure what cleanest/least redundant thing to do was.
I'll try your suggestion. :]